examples/Demo/Shared/Pages/DataGrid/Examples/DataGridMultiSelect.razor

```diff
@@ -4,16 +4,20 @@
     <FluentCheckbox @bind-Value="@UseSelectedItems"
                     @bind-Value:after="@(() => ResetSelectItems())"
                     Label="Use `SelectedItems` property" />
+    <FluentCheckbox @bind-Value="@RestrictToCheckbox"
+                    @bind-Value:after="@(() => ResetSelectItems())"
+                    Label="Use `RestrictToCheckbox` property" />
 </FluentStack>
 
 @if (UseSelectedItems)
 {
     @* Sample using SelectedItems  *@
     <div>Using SelectedItems</div>
 
-    <FluentDataGrid Items="@People" ShowHover="true" TGridItem="Person">
+    <FluentDataGrid Items="@People" ShowHover="true" TGridItem="Person" OnRowClick="@(row => Console.WriteLine($"Row {row.RowIndex} clicked"))" OnCellClick="@(cell => Console.WriteLine($"Cell {cell.GridColumn} clicked"))">
```

Comment:
You must also add these two events to the second `FluentDataGrid`. If not, no results will be displayed in the demo example.

Reply:
And could you align these attributes to avoid too long lines?

```xml
<FluentDataGrid Items="@People" ShowHover="true" TGridItem="Person"
                OnRowClick="@(row => Console.WriteLine($"Row {row.RowIndex} clicked"))"
                OnCellClick="@(cell => Console.WriteLine($"Cell {cell.GridColumn} clicked"))">
```